.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
```

Comment:
please also add `sudo: false` to this file. That will cause Travis to use docker containers, which start much faster.